### DISCUSSION TITLE
Detailed Routing Missing Guides

### DISCUSSION THREAD

**Discussion by kareefardi** [The-OpenROAD-Project/OpenROAD#6179] [Q&A]
DRT behavior seems to have been updated such that it only routes the nets the are covered by a guides file. Is there any adverse implications to this change? On the top of my head, GRT sometimes fail to cover every net (in case of allowed congestion? - can't really recall when this happens might not be even true). If that's the case the flow will end up partially routing the design.

**Comment by maliberty** [comment]
I don't know of a case where GRT fails to cover nets.  If you see a specific example please file an issue.

**Reply by eder-matheus** [reply]
That's right, GRT should never let a net without guides, except in the cases where the net has only 1 pin. If this happens, it's definitely a bug.

**Reply by kareefardi** [reply]
So I was talking about this warning:

```
[GRT-0026] Missing route to pin max_length1254/A.
```
Not sure what it implies but I assumed that it is related to some missing guides information. Is this warning still relevant and does it has anything to do with missing guides?

**Reply by eder-matheus** [reply]
This is related to the parasitics estimation from the route guides. We shouldn't have this warnings either, but it doesn't affect DRT. It is more about the topology and connecivity of the guides, which DRT doesn't care about.

**Reply by kareefardi** [reply]
Ok thanks. I will test if this warning persists in latest OpenROAD and file a separate issue if needed.
